specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,479 @@
+{
+  "openapi": "3.1.0",
+  "info": {
+    "title": "Open Tool Calling Standard",
+    "version": "1.0.0",
+    "description": "A standard for calling tools (functions or services) in a distributed system.",
+    "license": {
+      "name": "MIT",
+      "url": "https://opensource.org/licenses/MIT"
+    },
+    "contact": {
+      "name": "Arcade.dev",
+      "url": "https://arcade.dev",
+      "email": "dev@arcade.dev"
+    }
+  },
+  "paths": {
+    "/health": {
+      "get": {
+        "operationId": "health-check",
+        "summary": "Check health",
+        "description": "Checks the health status of the server.",
+        "security": [{}],
+        "responses": {
+          "200": {
+            "description": "Server is healthy."
+          },
+          "400": {
+            "description": "Server is unhealthy and cannot accept requests."
+          },
+          "500": {
+            "description": "Server has errored and cannot accept requests."
+          }
+        }
+      }
+    },
+    "/tools": {
+      "get": {
+        "operationId": "list-tools",
+        "summary": "List tools",
+        "description": "Returns a list of tool definitions.",
+        "security": [
+          {
+            "bearerAuth": []
+          },
+          {}
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of available tools.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "$schema": {
+                      "type": "string",
+                      "format": "uri"
+                    },
+                    "tools": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/ToolDefinition"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/execute": {
+      "post": {
+        "operationId": "execute-tool",
+        "summary": "Execute tool",
+        "description": "Executes a tool with the given parameters.",
+        "security": [
+          {
+            "bearerAuth": []
+          },
+          {}
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "properties": {
+                  "$schema": {
```

Comment:
* is it possible to inline schema as a field into ExecuteToolRequest and make it optional?
* server can use it if it's provided, and otherwise do the hard work of inferring what it is or just assuming it's the latest schema.

This optimizes for the client code (and ability to interact w/ the server using simple CURL requests without needing to specify a schema URI)

Reply:
I considered putting `$schema` inside `ExecuteToolRequest` but I wanted to have a clean separation: It is the _route's payload_ that must conform to a particular schema (version of this spec), not necessarily the object itself.

> This optimizes for the client code (and ability to interact w/ the server using simple CURL requests without needing to specify a schema URI)

Good point. I added a `required` block to make it explicit that `$schema` is optional, and added a note to the RFC describing what a server must do if the schema is not specified (assume latest)